Jenkinsfile

```diff
@@ -419,14 +436,15 @@ pipeline {
         axes {
           axis {
             name 'CASSANDRA_VERSION'
-            values '3.11',    // Current Apache Cassandra
-                   '4.0',     // Development Apache Cassandra
-                   'dse-5.1.35', // Legacy DataStax Enterprise
-                   'dse-6.8.30' // Development DataStax Enterprise
+            values '3.11',    // Previous Apache Cassandra
+                   '4.1.0',    // Previous Apache Cassandra 
+                   '5.0-beta1', // Current Apache Cassandra 
+                   'dse-6.9.0', // Current DataStax Enterprise
+                   'hcd-1.0.0' // HCD
```

Comment:
5.0 does not work because 5.0 build from source needs Java  17, while binary release 5.0.0 has `memtable` configuration problem. We have to use `5.0-beta1` first before `5.0.1` is out.

Reply:
The vote for 5.0.1 was just completed today.  Let's wait until 5.0.1 is available and then move to it; I like that a whole lot better than throwing out something now knowing we're going to have to revisit it almost immediately.